scalabel/label/from_mot.py

```diff
@@ -91,6 +89,37 @@ def parse_annotations(ann_filepath: str) -> Dict[int, List[Label]]:
     return outputs
 
 
+def save_split_files(split_index: int, orig_ann_path: str) -> None:
+    """Save out half train / val gt annotations in original format.
+
+    Note: frame index starts with 1 in this file (0 in scalabel format).
+    Hence, frame_id == frameIndex + 1.
+    """
+    train_lines, val_lines = [], []
+    for line in load_file_as_list(orig_ann_path):
+        gt = line.strip().split(",")
+        frame_id, _ = map(int, gt[:2])
+        if frame_id <= split_index:
+            train_lines.append(line)
+        else:
+            gt[0] = str(frame_id - split_index)
+            val_lines.append(",".join(gt) + "\n")
+
+    with open(
+        orig_ann_path.replace("gt.txt", "gt_half_train.txt"),
```

Comment:
Maybe a more generalizable way is `orig_ann_path.replace(".txt", "_half_train.txt")` to handle other `gt.txt` names? Although in MOT I know it's called `gt.txt`